### ISSUE TITLE
Set up Jenkins pipeline to create build artifacts

### ISSUE THREAD

**Issue by TrevCraw** [OpenLiberty/liberty-tools-vscode#143]
Jenkins should be triggered to create build artifacts when PRs are opened and when PRs are merged into the `main` branch.
The resulting artifact from this build will be the `.vsix` file, which is created by the command `vsce package`.
Node.js is a requirement for the build environment. The latest version of Node.js and NPM should work.
Commands that must be run prior to the `vsce package` command are `npm install` and `npm run compile`.

Once the `liberty-ls-prototype` branch is merged into `main`, there will be a build dependency on the Liberty LS, which can be pulled from Sonatype.
